ts/node/migration/sessionMigrations.ts

```diff
@@ -1854,6 +1886,58 @@ function updateToSessionSchemaVersion32(currentVersion: number, db: BetterSqlite
   console.log(`updateToSessionSchemaVersion${targetVersion}: success!`);
 }
 
+function updateToSessionSchemaVersion33(currentVersion: number, db: BetterSqlite3.Database) {
+  const targetVersion = 33;
+  if (currentVersion >= targetVersion) {
+    return;
+  }
+
+  console.log(`updateToSessionSchemaVersion${targetVersion}: starting...`);
+  db.transaction(() => {
+    db.exec(`ALTER TABLE ${CONVERSATIONS_TABLE} ADD COLUMN blocksSogsMsgReqsTimestamp INTEGER;`);
+
+    const loggedInUser = getLoggedInUserDuringMigration(db);
+    const ourConversation = getLoggedInUserConvoDuringMigration(db);
+
+    if (!ourConversation || !loggedInUser) {
+      // no user loggedin was empty. Considering no users are logged in
+      writeSessionSchemaVersion(targetVersion, db);
+      return;
+    }
+    // a user is logged in, we want to enable the 'inbox' polling for sogs, only if the current userwrapper for that field is undefined
+    const { privateEd25519 } = loggedInUser;
+
+    // Get existing config wrapper dump and update it
+    const userConfigWrapperDump = db
+      .prepare(`SELECT * FROM ${CONFIG_DUMP_TABLE} WHERE variant = 'UserConfig';`)
+      .get() as Record<string, any> | undefined;
+
+    if (!userConfigWrapperDump) {
+      writeSessionSchemaVersion(targetVersion, db);
+      return;
+    }
+    const userConfigData = (userConfigWrapperDump as any).data as Uint8Array | null;
+    const userProfileWrapper = new UserConfigWrapperNode(privateEd25519, userConfigData);
+
+    let blindedReqEnabled = userProfileWrapper.getEnableBlindedMsgRequest();
+
+    // if the value stored in the wrapper is undefined, we want to have blinded request enabled
+    if (isNil(blindedReqEnabled)) {
+      // this change will be part of the next ConfSyncJob (one is always made on app startup)
+      userProfileWrapper.setEnableBlindedMsgRequest(true);
```

Comment:
Of cool! So this means we can do things after the migration. Although I imagine it's not safe for multiple operations?